frontend/src/components/ImplementationReportView/DialectCompliance.tsx

```diff
@@ -42,7 +42,18 @@ const DialectCompliance: React.FC<{
               .map(([dialectName, result], index) => {
                 return (
                   <tr key={index}>
-                    <td>{Dialect.forPath(dialectName).uri}</td>
+                    <td>
+                      {Dialect.forPath(dialectName).uri}{" "}
+                      <img
+                        width={"auto"}
```

Comment:
Is defining width & height necessary?

Reply:
I tried to remove height and width parameters and UI is working as intended.